Makefile

```diff
@@ -234,8 +235,18 @@ worker-key-pair: ca
 	rm /etc/osbuild-composer/worker-csr.pem
 
 .PHONY: unit-tests
+.ONESHELL:
 unit-tests:
-	go test -race ./...
+	go test -race -covermode=atomic -coverprofile=coverage.txt -coverpkg=$$(go list ./... | grep -v rpmmd/test$ | tr "\n" ",") ./...
+	# sub module is not tested automatically
+	pushd pkg/splunk_logger
+	go test -race -covermode=atomic -coverprofile=../../coverage_splunk_logger.txt -coverpkg=$$(go list ./... | grep -v rpmmd/test$ | tr "\n" ",") ./...
+	popd
+
+.PHONY: coverage-report
+coverage-report: unit-tests
```

Comment:
This is nice - I think it's would slightly nicer to split it into it's own commit though.